libafl/src/feedbacks/mod.rs

```diff
@@ -438,6 +467,22 @@ where
         let b = second.is_interesting(state, manager, input, observers, exit_kind)?;
         Ok(a || b)
     }
+    #[cfg(feature = "track_hit_feedbacks")]
+    fn last_result(first: &A, second: &B) -> Option<bool> {
+        Some(
+            first.last_result().expect("should have run")
+                || second.last_result().expect("should have run"),
+        )
+    }
+    #[cfg(feature = "track_hit_feedbacks")]
+    fn get_hit_feedbacks(first: &A, second: &B, list: &mut Vec<String>) {
+        if first.last_result().expect("should have run") {
+            first.get_hit_feedbacks(list);
+        }
+        if second.last_result().expect("should have run") {
```

Comment:
For eager_or the second part should not be considered if the first part is `true`

Reply:
Ah well, not sure, maybe it should?
Let's keep it, even though it doesn't make a difference for the result, actually

Reply:
Even if it doesn't matter for the result, it's still a feedback that was run and was interesting so IMO it makes sense to add it.

Reply:
_however_ it did not contribute to the final outcome. It's more of fundamental question what to do.
I mean in general, I agree, but in this case:
`feeback_or!(true, feeback_and!(false, ThisFeedback))` it would not be counted, even though it's true as well, so...

Reply:
Good catch. Yeah I would remove it then, cause thinking about it, it would make sense to know *which* Feedback from the OR led to it's acceptance.

Reply:
I'll also add a note.